pkg/filters/data.go

```diff
@@ -37,6 +116,17 @@ func (af *DataFilter) Filter(eventID events.ID, data []trace.Argument) bool {
 		return true
 	}
 
+	// don't Filter this special case
```

Comment:
What do you think of checking a `kernelFilter` flag instead (it could be set on `Parse`)?

Reply:
Good ideia. I've called `af.EnableKernelFilter(id, dataName)` from `Parse`. The `EnableKernelFilter` should enable the filter in kernel for some events. After that we just need to check on `Filter` is the flag for a specific `eventID` was set.